androidsdk/src/main/java/com/wootric/androidsdk/SurveyValidator.java

```diff
@@ -44,17 +66,17 @@ public void validate() {
         Boolean firstDelay = firstSurveyDelayPassed();
         Boolean lastSeen = lastSeenDelayPassed();
 
-        Log.v(TAG, "IS SURVEY IMMEDIATELY ENABLED: " + immediate);
-        Log.v(TAG, "WAS RECENTLY SURVEYED: " + recently);
-        Log.v(TAG, "FIRST SURVEY DELAY PASSED: " + firstDelay);
-        Log.v(TAG, "LAST SEEN DELAY PASSED: " + lastSeen);
+        Log.d(TAG, "IS SURVEY IMMEDIATELY ENABLED: " + immediate);
+        Log.d(TAG, "WAS RECENTLY SURVEYED: " + recently);
+        Log.d(TAG, "FIRST SURVEY DELAY PASSED: " + firstDelay);
+        Log.d(TAG, "LAST SEEN DELAY PASSED: " + lastSeen);
 
         if (immediate || !recently || firstDelay || lastSeen) {
-            Log.v(TAG, "Needs survey. Will check with server.");
+            Log.d(TAG, "Needs survey. Will check with server.");
             checkEligibility();
         }
         else {
-            Log.v(TAG, "Doesn't need survey. Will not check with server.");
+            Log.d(TAG, "Doesn't need survey. Will not check with server.");
```

Comment:
Verbose and Debug. On Android Studio you can filter logs between Verbose, Debug, Info, Warn, Error and Assert. I like to use Debug over Verbose. Verbose shows A LOT of output from the OS.